web-marketplace/src/components/organisms/Order/Order.MakeAnonymous.tsx

```diff
@@ -0,0 +1,71 @@
+import { ChangeEvent, useState } from 'react';
+
+import { EditButtonIcon } from 'web-components/src/components/buttons/EditButtonIcon';
+import { TextButton } from 'web-components/src/components/buttons/TextButton';
+
+interface MakeAnonymousProps {
+  value: string;
+  onChange: (isAnonymous: string) => void;
+  name?: string;
+  ariaLabel?: string;
+  className?: string;
+}
+
+export const MakeAnonymous = ({
+  value,
+  onChange,
+  name = '',
+  ariaLabel = 'Make Anonymous select',
+  className = '',
+}: MakeAnonymousProps) => {
+  const [editable, setEditable] = useState(false);
+  const [isAnonymous, setIsAnonymous] = useState(value);
+
+  const toggleEditable = () => {
+    setEditable(!editable);
+  };
+
+  const handleOnChange = (e: ChangeEvent<HTMLSelectElement>) => {
+    e.preventDefault();
+    setIsAnonymous(e.target.value);
+  };
+
+  const handleOnUpdate = () => {
+    onChange(isAnonymous);
+    toggleEditable();
+  };
+
+  return (
+    <>
+      {editable ? (
+        <div
+          className={`w-full flex justify-between [@media(max-width:340px)]:flex-col [@media(max-width:340px)]:mb-20 sm:flex-row items-center [@media(max-width:340px)]:items-start h-[47px] ${className}`}
+        >
+          <select
+            className="w-[120px] py-10 px-10 border border-gray-300 text-base font-normal font-['Lato'] flex items-center"
+            value={isAnonymous}
+            onChange={handleOnChange}
+            aria-label={ariaLabel}
+            name={name}
+            autoFocus
+          >
+            <option value="Yes">Yes</option>
+            <option value="No">No</option>
```

Comment:
I have added the translations although I wasn't sure because we don't need them now because this component is not going to be fully implemented at this stage.

Reply:
I believe that as long as we create new components, we should add translations everywhere we should, even though it's not complete yet, because even if we'll work again on this component, we might just forget about the texts to translate